internal/install/application_configuration.go

```diff
@@ -66,7 +68,7 @@ func (ac *ApplicationConfiguration) StackImageRefs(version string) ImageRefs {
 
 // Configuration function returns the elastic-package configuration.
 func Configuration() (*ApplicationConfiguration, error) {
-	configPath, err := configurationDir()
+	configPath, err := locations.ConfigurationDir()
```

Comment:
Would it make sense to use `NewLocationManager()` here and then call the `RootDir()` method on the new `LocationManager` instance? This would allow us to unexport `locations.ConfiguratoinDir()` and uniformly access configuration locations throughout the codebase exclusively via a `LocationManager` instance.